Dockerfile

```diff
@@ -7,7 +7,10 @@ RUN rm -f /mysqld_exporter/mysqld_exporter
 RUN CGO_ENABLED=0 GOOS="$TARGETOS" GOARCH="$TARGETARCH" make -C /mysqld_exporter build
 
 FROM golang:alpine
+RUN apk --no-cache add curl
+RUN apk --no-cache add jq
 COPY --from=build /mysqld_exporter/mysqld_exporter /bin/mysqld_exporter
 EXPOSE 9104
 USER nobody
+WORKDIR /
```

Comment:
is this required for anything or just a nice to have?

Reply:
Oops. It's a non-functional change I had teed up in my work tree in case I ever needed a dummy change to force the CI through another loop. I actually like it being in because otherwise the working directory is `/go` which is harmless but kind of nonsense.